content/blogs/newsletter-104.md

```diff
@@ -0,0 +1,124 @@
+---
+title:       "Newsletter 104 - April/May news"
+author:      "GeoB99"
+date:        2024-04-22
+tags:        [ newsletter ]
+---
+
+Greetings from the ReactOS Team!
+This year, we will start with a short summary highlighting ongoing work on the project as well as announcing the selection of our brand new developers.
+We will also talk about the current situation with symmetric multiprocessing (SMP) support in ReactOS.
+
+## Development team enlargement
+Over the last decade, the number of people in the project team base has not changed much.
+While development continued and still continues as usual, the development team hadn't seen any increase in new developers.
+In fact, some of the existing core developers became inactive for quite a while, some of whom were maintainers of important core modules of the project.
```

Comment:
Suggestion:
In fact, some of the existing core developers became inactive some time ago and some of these were maintainers of important core modules of the project.